dotcom-rendering/src/web/components/OnwardsData.tsx

```diff
@@ -75,7 +74,7 @@ export const OnwardsData = ({
 		return (
 			<div css={[minHeight, revealStyles]} className="onwards">
 				<div className="pending">
-					<Container
+					<Carousel
```

Comment:
We used to support the `Container` being passed in as a prop but now we always render the same component (Carousel) there's no longer any need to do this